### ISSUE TITLE
Still maintained? And if yes is there a chance a 1.0 will be released soon?

### ISSUE THREAD

**Issue by dominikzogg** [azjezz/hack-routing#3]
Would love to release https://github.com/chubbyphp/chubbyphp-framework-router-hack-routing if there is a stable version of this library.

**Comment by azjezz** [comment]
Yes, i'm planning on cleaning up the repository and releasing 1.0, however, i didn't have time to do so yet, hopefully i can get to it soon :)
